5.11/5.11.2/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 5.11.2
+Сделать объект с необходимыми методами.
+Главный метод этого объекта принимает число (количество секунд, которое прошло после 2000 года)
+и возвращает строку даты в формате: "2002г. сентябрь 07"
+<script>
+	const dateInfo = {
+		getDateString: function (seconds) {
+			const date = new Date(2000, 0, 1);
+			const newDateMilliseconds = date.getTime() + (seconds * 1000);
```

Comment:
just `newMilliseconds` would be ok.
And you don't need round brackets for operations with multiplication